roosevelt.js

```diff
@@ -301,6 +303,7 @@ module.exports = function (params) {
     httpsServer: httpsServer,
     expressApp: app,
     initServer: initServer,
-    startServer: startServer
+    startServer: startServer,
+    shutdownServer: gracefulShutdown
```

Comment:
change this to `stopServer`

and update the "Roosevelt object members" table in the README to be current; so that all objects on this list are reflected there